index.html

```diff
@@ -378,6 +378,106 @@ <h1 id="contact-me">Contact me</h1>
+                          clip-rule="evenodd"
+                          d="M5 12C5 8.13401 8.13401 5 12 5C12.5523 5 13 4.55228 13 4C13 3.44772 12.5523 3 12 3C7.02944 3 3 7.02944 3 12C3 16.9706 7.02944 21 12 21C16.9706 21 21 16.9706 21 12C21 11.4477 20.5523 11 20 11C19.4477 11 19 11.4477 19 12C19 15.866 15.866 19 12 19C8.13401 19 5 15.866 5 12ZM16 3C15.4477 3 15 3.44772 15 4C15 4.55228 15.4477 5 16 5H17.5858L11.2929 11.2929C10.9024 11.6834 10.9024 12.3166 11.2929 12.7071C11.6834 13.0976 12.3166 13.0976 12.7071 12.7071L19 6.41421V8C19 8.55228 19.4477 9 20 9C20.5523 9 21 8.55228 21 8V4C21 3.44772 20.5523 3 20 3H16Z"
+                          fill="#6070FF"
+                        />
+                      </svg>
+                    </a>
+                    <a
+                      id="work-modal-source"
+                      href="#"
+                      class="button button-enabled flex"
+                      >See source
+                      <svg
+                        width="24"
+                        height="24"
+                        viewBox="0 0 24 24"
+                        fill="none"
+                        xmlns="http://www.w3.org/2000/svg"
+                      >
+                        <g clip-path="url(#clip0_39_522)">
+                          <path
+                            d="M20.9989 9.46114C20.9989 8.32124 20.6329 7.25043 19.9342 6.31779C20.2004 5.41969 20.2669 4.27979 20.2004 2.96718C20.1671 2.41451 19.7345 2 19.2022 2C18.9028 2 16.3741 2.03454 14.9101 3.38169C13.6458 3.1399 12.315 3.1399 11.0174 3.38169C9.58667 2.03454 7.05802 2 6.7253 2C6.19296 2 5.76042 2.41451 5.72715 2.96718C5.62734 4.27979 5.72715 5.41969 5.99333 6.31779C5.29462 7.28497 4.92863 8.35579 4.92863 9.46114C4.92863 11.8791 6.7253 14.0553 9.45358 15.0915C9.35377 15.2988 9.28723 15.5406 9.22068 15.7824C6.32604 15.4715 4.9619 12.7427 4.89536 12.639C4.66246 12.1209 4.06357 11.9136 3.56449 12.19C3.06541 12.4318 2.86578 13.0535 3.13196 13.5717C3.1985 13.7444 5.02845 17.4404 9.05432 17.8549V20.9637C9.05432 21.5509 9.48686 22 10.0525 22H15.875C16.4406 22 16.8732 21.5509 16.8732 20.9637V16.8532C16.8732 16.2314 16.7401 15.6442 16.5072 15.1261C19.2022 14.0553 20.9989 11.9136 20.9989 9.46114Z"
+                            fill="#6070FF"
+                          />
+                        </g>
+                        <defs>
+                          <clipPath id="clip0_39_522">
+                            <rect
+                              width="18"
+                              height="20"
+                              fill="white"
+                              transform="translate(3 2)"
+                            />
+                          </clipPath>
+                        </defs>
+                      </svg>
+                    </a>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] You have done a great job styling your modal popup. It is remaining with something very small now. Kindly note that you are supposed to stick to the chosen Figma design as much as possible for margins, padding and fonts. Currently your images are too large which makes them take up a lot of space and the text you added is also too long. I recommend that you try to cut out some of the unwanted text and stick to the one found in the Figma template.

**This is how your project currently looks like:**

**1. Image being too large**

![image](https://user-images.githubusercontent.com/86318284/180283005-630caa1a-f635-49f4-9d8f-ac7889119e33.png)

**2. The text is too long:**

![image](https://user-images.githubusercontent.com/86318284/180283339-a3b61003-8075-4930-b0a7-661b5b34777a.png)


**3. You do not stick to the project margins and padding:**

![image](https://user-images.githubusercontent.com/86318284/180283244-2c6a4ab0-0a95-4962-8f9f-20f99a0a673e.png)

**4. The buttons and the technologies used are not properly aligned**

![image](https://user-images.githubusercontent.com/86318284/180283485-b4bf8b81-b4ae-4f5c-9cb5-37320e889451.png)


**This is the desired desktop design modal popup"**

![image](https://user-images.githubusercontent.com/86318284/180283689-acad5c40-1c4e-4bbb-87f0-9ae703fa131a.png)